.github/workflows/rust.yml

```diff
@@ -8,4 +8,4 @@ jobs:
     steps:
     - uses: actions/checkout@v1
     - run: rustup update stable
-    - run: make ci
+    - run: make ci
```

Comment:
```suggestion
    - run: make ci

```

Reply:
Reminder to add a newline (not CLRF!) to the end of this so that it doesn't change with this PR.